man/tmpfiles.d.xml

```diff
@@ -340,7 +345,17 @@ L     /tmp/foobar -    -    -     -   /dev/null</programlisting>
           parameter. Note that lines of this type do not influence the
           effect of <varname>r</varname> or <varname>R</varname>
           lines. Lines of this type accept shell-style globs in place
-          of normal path names.  </para></listitem>
+          of normal path names.  </para>
+
+          <para>If the age argument is specified in this line, the path
+          will still be subject to time based cleanup, but the actual
+          behavior depends on the type of the path: Similar to <varname>C</varname>,
+          only the contents of a directory will be cleaned, while those
+          pointing to single files will be skipped. It should be noted
+          that glob expansion doesn't happen when the type of path is
+          checked during cleanup, so paths that contain glob characters
+          are free from being cleaned, since there should not be directories
+          whose name contain asterisks.</para></listitem>
```

Comment:
Hmm, this sounds wrong. If we are really cleaning files listed in `x`, then it's just a plain bug.

Reply:
Yeah, I didn't expect messy things like this to become part of the documentation, so I add a bit of humor to the last sentence.

Reply:
#20641 changes the code not do to that…